python/cudf/cudf/tests/test_copying.py

```diff
@@ -280,6 +280,7 @@ def test_series_zero_copy_cow_on():
         # modify `s` and will leave rest of the copies
         # untouched.
 
+        assert_eq(s.to_numpy(), np.array([10, 10, 10, 4, 5]))
```

Comment:
Are these changes sufficient with a column with child columns (like string or list)?

Reply:
I don't think we will be able to hit this case in other columns, because of the mutability aspect.

Reply:
Gotcha, sounds good.